### DISCUSSION TITLE
What happened to application file of weblate project?

### DISCUSSION THREAD

**Discussion by Blueberryy** [WeblateOrg/weblate#6766] [Q&A] [answered]
It's locked and i can't push new translation

**Comment by nijel** [comment, marked as answer]
Either one of the admins has locked it, or there was an error in pushing or pulling changes. That should be visible at the lock message. 

It's hard to tell without having additional information.

**Reply by Blueberryy** [reply]
Where can i see the lock message?

**Reply by nijel** [reply]
On the same page where you cannot push? The history might be useful as well in case it was locked manually.

**Reply by Blueberryy** [reply]
It says that it was locked by you and for all languages. I guess you're making some update for them

**Reply by nijel** [reply]
Ah, you are talking about https://hosted.weblate.org/projects/weblate/application/? It would be good to mention it somewhere :-). It is now unlocked, it was caused by issue in our messages update workflow.
